diff --git a/papermill/tests/test_utils.py b/papermill/tests/test_utils.py
index 4e24ce7..9edf861 100644
--- a/papermill/tests/test_utils.py
+++ b/papermill/tests/test_utils.py
@@ -55,6 +55,6 @@ def test_chdir():
     with TemporaryDirectory() as temp_dir:
         with chdir(temp_dir):
             assert Path.cwd() != old_cwd
-            assert Path.cwd() == Path(temp_dir)
+            assert Path.cwd().resolve() == Path(temp_dir).resolve()
 
     assert Path.cwd() == old_cwd
